int8 drone_id
crazyflie_trajectory_manager/LoadTrajectoryPayload payload
---
bool proxy_success
bool result